When querier is detected, static MDB entries cannot be deleted, fix this.

diff --git a/net/bridge/br_mdb.c b/net/bridge/br_mdb.c
index c0b8bbc..50b5e76 100644
--- a/net/bridge/br_mdb.c
+++ b/net/bridge/br_mdb.c
@@ -535,15 +535,9 @@ static int __br_mdb_del(struct net_bridge *br, struct br_mdb_entry *entry)
 	ip.proto = entry->addr.proto;
 	ip.vid = entry->addr.vlan_id;
 	if (ip.proto == htons(ETH_P_IP)) {
-		if (timer_pending(&br->ip4_querier.timer))
-			return -EBUSY;
-
 		ip.u.ip4 = entry->addr.u.ip4;
 #if IS_ENABLED(CONFIG_IPV6)
 	} else {
-		if (timer_pending(&br->ip6_querier.timer))
-			return -EBUSY;
-
 		ip.u.ip6 = entry->addr.u.ip6;
 #endif
 	}
